create_pnor_image.pl

```diff
@@ -131,6 +131,9 @@
 $build_pnor_command .= " --binFile_SECBOOT $scratch_dir/secboot.bin.ecc";
 $build_pnor_command .= " --binFile_VERSION $openpower_version_filename";
 $build_pnor_command .= " --binFile_IMA_CATALOG $scratch_dir/ima_catalog.bin.ecc";
+if ($release eq "p9"){
```

Comment:
This case is taken care below on line 140 (if p8, else).
I do think that we should be more specific though, so maybe move the p9 check to below the p8 one?